include/scenefx/types/wlr_scene.h

```diff
@@ -437,6 +439,18 @@ void wlr_scene_rect_set_clipped_region(struct wlr_scene_rect *rect,
  */
 void wlr_scene_rect_set_color(struct wlr_scene_rect *rect, const float color[static 4]);
 
+/**
+* Sets the whether or not the buffer should render backdrop blur
+*/
+void wlr_scene_rect_set_backdrop_blur(struct wlr_scene_rect *rect,
+		bool enabled);
+
+/**
+* Sets the whether the backdrop blur should use optimized blur or not
```

Comment:
Same here

Reply:
This was a copy paste, looks like we need to update the buffer comments too 😅 nice catch